packages/components/src/stories/data/scenario.stats.json

```diff
@@ -0,0 +1,128 @@
+[
```

Comment:
In fact, this "stats" is just frequently occurring functions. So, I'd suggest making this data an object that has a single key (for now) like `functions` - anticipating that we may add more stats in the future.

Reply:
Okay, makes sense. I passed this off to @typaulhus for now, but I'll change this before it gets merged.